AsyncDisplayKitTests/ASTableViewTests.m

```diff
@@ -155,28 +155,34 @@ - (void)testReloadData
   [tableView reloadData];
   
   for (int i = 0; i < NumberOfReloadIterations; ++i) {
-    UITableViewRowAnimation rowAnimation = (arc4random_uniform(1) == 0 ? UITableViewRowAnimationMiddle : UITableViewRowAnimationNone);
-    
-    BOOL animatedScroll               = (arc4random_uniform(1) == 0 ? YES : NO);
-    BOOL reloadRowsInsteadOfSections  = (arc4random_uniform(1) == 0 ? YES : NO);
-    BOOL letRunloopProceed            = (arc4random_uniform(1) == 0 ? YES : NO);
-    BOOL useBeginEndUpdates           = (arc4random_uniform(2) == 0 ? YES : NO);
+    UITableViewRowAnimation rowAnimation = (arc4random_uniform(2) == 0 ? UITableViewRowAnimationMiddle : UITableViewRowAnimationNone);
+    BOOL animatedScroll               = (arc4random_uniform(2) == 0 ? YES : NO);
```

Comment:
Methodology for changing the frequencies of these?  It seems fine to do, but can you bring the changes to ASTableViewStressTest as well?  Sadly there is code duplication between them, but not complete duplication because the test method needs to run in isolation of other things.  The editable data source of the sample project is much more flexible to fully test, and has found issues that this test cannot.  Similarly we need to move to ASCollectionViewStressTest soon and also thrash supplementary views, which are currently known to be crashing but untested.